coresdk/src/backend/gpio_driver.cpp

```diff
@@ -0,0 +1,140 @@
+/***********************************************
+ * XQuestCode || Aditya Parmar
+ * © 2024 Aditya Parmar. All Rights Reserved.
+ * This file is part of the SplashKit Core Library.
+ * Use https://abyz.me.uk/rpi/pigpio/pdif2.html for reference
+ ***********************************************/
+
+#include "gpio_driver.h"
+
+#include <string>
+#include <iostream>
+#include <cstdlib> // Add this line to include the necessary header for the exit() function
+
+#ifdef RASPBERRY_PI
```

Comment:
I would aim to have this around the entire file - not in each function. Then the frontend code can have this in each function...